source/registry/selectors/context.json

```diff
@@ -0,0 +1,60 @@
+{
+  "@context": {
+    "@version":1.1,
+    "iiif_selectors": "http://iiif.io/api/registry/selectors#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+
+    "ImageApiSelector": {
+      "@id": "iiif_selectors:ImageApiSelector",
+      "@context": [
+        "http://iiif.io/api/image/#",
```

Comment:
I think this should be defined above as `"iiif_image": "http://iiif.io/api/image/#"` and not listed in this `@context`